src/components/validators/project-name-available-validator.js

```diff
@@ -6,6 +6,9 @@
       return {
         restrict: 'A',
         require: 'ngModel',
+        scope: {
+          organizationId: "="
```

Comment:
This is fine, `=` is a two way binding where this could be one way binding but I'm fine with it :).